src/events/local-storage-backed-named-event-queue.ts

```diff
@@ -47,7 +47,10 @@ export default class LocalStorageBackedNamedEventQueue<T> implements NamedEventQ
     if (this.eventKeys.length === 0) {
       return undefined;
     }
-    const eventKey = this.eventKeys.shift()!;
+    const eventKey = this.eventKeys.shift();
+    if (eventKey === undefined) {
```

Comment:
why not simply `if (!eventKey)`?

Reply:
Yeah good call, this was an autofix by the IDE 😅